### PR TITLE
chore(website): Add docs on OpenTelemetry and Jaeger

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

File: website/pages/docs/advanced-topics/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+title: Monitoring CloudQuery
+description: Short walkthrough on how to use open telemetry to monitor CloudQuery Syncs
+---
+
+# Monitoring
+
+Monitoring CloudQuery can be done in a number of main ways:
+
+- Logging
+- Open-Telemetry
```
Comment (disq) [inline website/pages/docs/advanced-topics/monitoring.md:11]:
```suggestion
- OpenTelemetry
```

File: website/pages/docs/advanced-topics/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## OpenTelemetry (Preview)
+
+ELT workloads can be long running and sometimes it is necessary to understand better what calls taking the most time to potentially optimize those on the plugin side, ignore them or split them to a different workload.
+
+CloudQuery supports [OpenTelemetry](https://opentelemetry.io/) tracing out of the box and can be enabled easily via [configuration](../reference/source-spec).
+
+To collect traces you need a collector that supports OpenTelemetry protocol, for example [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/). For example you can use [Jaeger](https://opentelemetry.io/docs/instrumentation/go/exporters/#jaeger) to visualize and analyze traces.
+
+To start jaeger locally you can use docker:
```
Comment (disq) [inline website/pages/docs/advanced-topics/monitoring.md:25]:
```suggestion
To start Jaeger locally you can use Docker:
```

File: website/pages/docs/advanced-topics/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Logging
+- OpenTelemetry
+
+## Logging
+
+CloudQuery is utilizing structured [logging](../reference/cli/cloudquery) (in plain and json formats) which can be analyzed by local tools such as `jq`, `grep` and remote aggregations tools like `loki`, `datadog` or any other popular log aggregation that supports structured logging.
+
+## OpenTelemetry (Preview)
+
+ELT workloads can be long running and sometimes it is necessary to understand better what calls taking the most time to potentially optimize those on the plugin side, ignore them or split them to a different workload.
```
Comment (disq) [inline website/pages/docs/advanced-topics/monitoring.md:19]:
```suggestion
ELT workloads can be long running and sometimes it is necessary to better understand what calls are taking the most time; to potentially optimize those on the plugin side, ignore them or split them to a different workload.
```

File: website/pages/docs/advanced-topics/monitoring.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+# Monitoring
+
+Monitoring CloudQuery can be done in a number of main ways:
+
+- Logging
+- OpenTelemetry
+
+## Logging
+
+CloudQuery is utilizing structured [logging](../reference/cli/cloudquery) (in plain and json formats) which can be analyzed by local tools such as `jq`, `grep` and remote aggregations tools like `loki`, `datadog` or any other popular log aggregation that supports structured logging.
```
Comment (erezrokah) [inline website/pages/docs/advanced-topics/monitoring.md:15]:
```suggestion
CloudQuery utilizes structured [logging](../reference/cli/cloudquery) (in plain and JSON formats) which can be analyzed by local tools such as `jq`, `grep` and remote aggregations tools like `loki`, `datadog` or any other popular log aggregation that supports structured logging.
```

File: website/pages/docs/advanced-topics/monitoring.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## OpenTelemetry (Preview)
+
+ELT workloads can be long running and sometimes it is necessary to better understand what calls are taking the most time; to potentially optimize those on the plugin side, ignore them or split them to a different workload.
+
+CloudQuery supports [OpenTelemetry](https://opentelemetry.io/) tracing out of the box and can be enabled easily via [configuration](../reference/source-spec).
```
Comment (erezrokah) [inline website/pages/docs/advanced-topics/monitoring.md:21]:
```suggestion
CloudQuery supports [OpenTelemetry](https://opentelemetry.io/) tracing out of the box and can be enabled easily via [configuration](/docs//reference/source-spec).
```

We usually have absolute links so if we move this page to another location the link still works

File: website/pages/docs/advanced-topics/monitoring.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+```yaml
+kind: source
+spec:
+  name: "aws"
+  path: "cloudquery/aws"
+  version: "VERSION_SOURCE_AWS"
+  tables: ["aws_s3_buckets"]
+  destinations: ["postgresql"]
+  enabled: true
```
Comment (erezrokah) [inline website/pages/docs/advanced-topics/monitoring.md:45]:
```suggestion
```

Is `enabled: true` part of the spec?

**Review by erezrokah — COMMENTED** [review]
Added a few comments

**Pull request merged**